src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import { required, minLength, email } from '../../components/Validation';
+import { customInput } from '../../components/fields';
+import './styles.css';
+
+function Login({ handleSubmit, disableSubmit }) {
+  return (
+    <form>
+      <h1> Welcome to Login </h1>
+      <Field name="email" component={customInput} type="text" label="Email" validate={[required, email]} />
+      <Field
+        name="password"
```

Comment:
create in `constants` a file named `fieldNames.js` and put there all the field names and the form name.

```
export const loginFields = {
    EMAIL: 'email',
    PASSWORD: 'password',
    FORM_NAME: 'login'
};
```